jaeger-core/src/main/java/com/uber/jaeger/propagation/TextMapCodec.java

```diff
@@ -40,16 +40,22 @@
 
   private static final PrefixedKeys keys = new PrefixedKeys();
 
-  private final String contextKey = SPAN_CONTEXT_KEY;
+  private String contextKey = SPAN_CONTEXT_KEY;
```

Comment:
Something like this would work:

````       
      private final String contextKey;

	  private final String baggagePrefix;

	  private final boolean urlEncoding;

	  public TextMapCodec(boolean urlEncoding) {
	    this.urlEncoding = urlEncoding;
	    this.contextKey = SPAN_CONTEXT_KEY;
	    this.baggagePrefix = BAGGAGE_KEY_PREFIX;
	  }
```